pkg/rulefmt/rulefmt_test.go

```diff
@@ -184,3 +186,120 @@ groups:
 	err1 := errs[1].(*Error).Err.node
 	require.NotEqual(t, err0, err1, "Error nodes should not be the same")
 }
+
+func TestError_Error(t *testing.T) {
+	tests := []struct {
+		name     string
+		Group    string
+		Rule     int
+		RuleName string
+		Err      WrappedError
```

Comment:
Instead of replicating `Error`'s fields here, I think it would be more concise if we just specified a complete `Error` object.

```suggestion
		Error *Error
```